sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClientIntegrationTest.java

```diff
@@ -1145,6 +1148,8 @@ void listTopics(HttpClient httpClient) {
 
     //endregion
 
+    //region Update Entity tests
```

Comment:
regions in Java 😅

Reply:
The test file is so long, it was the only way I could reasonably understand them. 😅